.gitignore

```diff
@@ -157,3 +157,10 @@ cython_debug/
 
```

Comment:
The test_data nifty files are 6.2MB each, this is unnecessarily big for the repo. 

I think we should shrink them to smaller sizes, which will:
- reduce repo/git size
- speed up unit tests

Reply:
Those are already downscaled 3d nifti files.  I'll try adding them to git LFS in the future, but at the moment I am at the end of my quota. 
Given that I/O is done by highly optimized C-classes, unit tests are not much faster with smaller files.

Reply:
Also, wouldn't the git history stay the same, even if I resize the files now? So the repo will remain the same size (possibly even grow) if I resize the images?